misk-prometheus/src/main/kotlin/misk/metrics/backends/prometheus/PrometheusMetrics.kt

```diff
@@ -32,8 +32,9 @@ internal class PrometheusMetrics @Inject internal constructor(
     name: String,
     help: String,
     labelNames: List<String>,
-    quantiles: Map<Double, Double>
-  ): Histogram = PrometheusHistogram(metricsV2.summary(name, help, labelNames, quantiles))
+    quantiles: Map<Double, Double>,
+    maxAgeSeconds: Long?
```

Comment:
```suggestion
    maxAgeSeconds: Long? = null,
```

Reply:
nit: add default value to avoid making a breaking change with the function signature